README.md

```diff
@@ -39,9 +29,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Database Microverse Project
 
 ## 📝 License
 
```

Comment:
- [OPTIONAL] The link of your MIT file is broken, So I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

Reply:
thank you, I will do it